columnflow/plotting/plot_util.py

```diff
@@ -227,10 +229,10 @@ def prepare_plot_config(
             "hist": h_mc_stack,
             "kwargs": {
                 "norm": mc_norm,
-                "label": mc_labels,
-                "color": mc_colors,
-                "edgecolor": mc_edgecolors,
-                "linewidth": [(0 if c is None else 1) for c in line_colors],
+                "label": mc_labels[::-1],
+                "color": mc_colors[::-1],
+                "edgecolor": mc_edgecolors[::-1],
+                "linewidth": [(0 if c is None else 1) for c in mc_colors[::-1]],
```

Comment:
Not strictly related to this PR, but it looks like this should actually be `mc_colors` instead of `line_colors`?